src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -128,11 +145,11 @@ const EVENT_LINKS: EventLink[] = [
   {
     linkRelation: ECHO,
     linkingEventId: [
-      TALENTS_EVOKER.SPIRITBLOOM_TALENT.id,
+      SPELLS.SPIRITBLOOM_SPLIT.id,
+      SPELLS.SPIRITBLOOM.id,
       SPELLS.LIVING_FLAME_HEAL.id,
       SPELLS.DREAM_BREATH_ECHO.id,
       TALENTS_EVOKER.VERDANT_EMBRACE_TALENT.id,
-      SPELLS.EMERALD_BLOSSOM_ECHO.id,
```

Comment:
Was this removed on purpose? If so, why?

Reply:
Readded it, oops